### DISCUSSION TITLE
0.25.3 -> 0.26.0 column_* api removed from Row

### DISCUSSION THREAD

**Discussion by neonphog** [rusqlite/rusqlite#1106] [Questions] [answered]
See [docs.rs/rusqlite/0.25.3](https://docs.rs/rusqlite/0.25.3/rusqlite/struct.Row.html) vs [docs.rs/rusqlite/0.26.0](https://docs.rs/rusqlite/0.26.0/rusqlite/struct.Row.html).

Was this intentional without a deprecation warning?

**Comment by trevyn** [comment, marked as answer]
Looks like you can do `row.as_ref().column_`* now instead:

https://github.com/rusqlite/rusqlite/pull/887/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759

**Comment by neonphog** [comment]
Awesome, I can use that in the mean time.
